books/views.py

```
@@ -249,7 +237,7 @@ class UserBookViewSet(viewsets.ModelViewSet):
     filter_backends = [DjangoFilterBackend, SearchFilter, OrderingFilter]
     filterset_fields = ["reading_status"]
     search_fields = ["book__title", "book__author__name", "reading_status"]
-    ordering_fields = ["date_added", "date_started", "date_finished", "book__title", "reading_status"]
+    ordering_fields = ["date_added", "book__title", "reading_status"]
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)